src/crimson/os/alienstore/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/rocksdb/include")
 
 add_library(alien::cflags INTERFACE IMPORTED)
 set_target_properties(alien::cflags PROPERTIES
-  INTERFACE_COMPILE_DEFINITIONS "WITH_SEASTAR;WITH_ALIEN"
+  INTERFACE_COMPILE_DEFINITIONS ""
```

Comment:
> Mere WITH_SEASTAR is enough to differentiate between "alien" targets to ones used by Crimson.

What about those pieces of `common` that are needed by both _alienized BlueStore_ and SeaStar-native parts of crimson **but in different variants**? Do the further commits in the PR tackle them?

Reply:
The compiled binary would the same with this patch.
Previously, any alienized target needed WITH_SEASTAR && !WITH_ALIEN.
Now that alienstore targets no longer defined with both, only WITH_SEASTAR is needed for the SeaStar-native parts.

Reply:
It would probably make more sense to replace WITH_SEASTAR to WITH_CRIMSON now (since alien_store actaully uses seastar but is not defined WITH_SEASTORE).